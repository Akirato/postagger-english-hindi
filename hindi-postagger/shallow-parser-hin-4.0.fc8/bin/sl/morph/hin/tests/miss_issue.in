1.	SrqMKalA	unk
2.	inasAna	unk
3.	AMKoM	unk
4.	yojanAbaxXa	unk
5.	PIsaxI	unk
6.	kAMca	unk
6.	kAzca	unk
7.	paripUrNawA	unk
8.	paScAwa	unk
9.	BAgoM	unk
10.	anUTI	unk
11.	surKiyoM	unk
12.	paracama	unk
13.	sabakA	unk
14.	sWAnAMwariwa	unk
15.	KAwmA	unk
16.	nAkAma	unk
17.	wahasa	unk
18.	nahasa	unk
19.	KuSagavAra	unk
20.	SOYla	unk
21.	kaSIxAkArI	unk
22.	raNaBUmi	unk
23.	piCadZepana	unk
24.	XImI	unk
25.	halakI	unk
26.	sWAyI	unk
27.	valavalA	unk
