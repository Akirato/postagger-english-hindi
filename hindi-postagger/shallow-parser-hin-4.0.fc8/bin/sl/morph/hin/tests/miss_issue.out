1	SrqMKalA	unk	<fs af='SrqMKalA,n,f,sg,3,d,0,0'>|<fs af='SrqMKalA,n,f,sg,3,o,0,0'>
2	inasAna	unk	<fs af='inasAna,n,m,sg,3,d,0,0'>|<fs af='inasAna,n,m,pl,3,d,0,0'>|<fs af='inasAna,n,m,sg,3,o,0,0'>
3	AMKoM	unk	<fs af='AMKa,n,f,pl,3,o,0,0'>
4	yojanAbaxXa	unk	<fs af='yojanAbaxXa,adj,any,any,,any,,'>
5	PIsaxI	unk	<fs af='PIsaxI,n,m,sg,3,d,0,0'>|<fs af='PIsaxI,n,m,pl,3,d,0,0'>|<fs af='PIsaxI,n,m,sg,3,o,0,0'>|<fs af='PIsaxI,adv,,,,,,'>
6	kAMca	unk	<fs af='kAMca,n,m,sg,3,d,0,0'>|<fs af='kAMca,n,m,pl,3,d,0,0'>|<fs af='kAMca,n,m,sg,3,o,0,0'>
7	kAzca	unk	<fs af='kAzca,n,m,sg,3,d,0,0'>|<fs af='kAzca,n,m,pl,3,d,0,0'>|<fs af='kAzca,n,m,sg,3,o,0,0'>
8	paripUrNawA	unk	<fs af='paripUrNawA,n,f,sg,3,d,0,0'>|<fs af='paripUrNawA,n,f,pl,3,d,0,0'>|<fs af='paripUrNawA,n,f,sg,3,o,0,0'>|<fs af='paripUrNawA,n,f,pl,3,o,0,0'>
9	paScAwa	unk	<fs af='paScAwa,adv,,,,,,'>|<fs af='paScAwa,psp,,,,,,'>
10	BAgoM	unk	<fs af='BAga,n,m,pl,3,o,0,0'>
11	anUTI	unk	<fs af='anUTA,adj,f,any,,any,,'>
12	surKiyoM	unk	<fs af='surKi,n,f,pl,3,o,0,0'>|<fs af='surKI,n,f,pl,3,o,0,0'>
13	paracama	unk	<fs af='paracama,n,m,sg,3,d,0,0'>|<fs af='paracama,n,m,pl,3,d,0,0'>|<fs af='paracama,n,m,sg,3,o,0,0'>
14	sabakA	unk	<fs af='saba,pn,any,pl,3,o,kA,kA' agr_gen='m'  agr_cas='d'  agr_num='sg'>
15	sWAnAMwariwa	unk	<fs af='sWAnAMwariwa,n,m,sg,3,d,0,0'>|<fs af='sWAnAMwariwa,n,m,pl,3,d,0,0'>|<fs af='sWAnAMwariwa,n,m,sg,3,o,0,0'>|<fs af='sWAnAMwariwa,adj,any,any,,any,,'>
16	KAwmA	unk	<fs af='KAwmA,n,m,sg,3,d,0,0'>
17	nAkAma	unk	<fs af='nAkAma,n,m,sg,3,d,0,0'>|<fs af='nAkAma,n,m,pl,3,d,0,0'>|<fs af='nAkAma,n,m,sg,3,o,0,0'>|<fs af='nAkAma,adj,any,any,,any,,'>
18	wahasa	unk	<fs af='wahasa,unk,,,,,,'>
19	nahasa	unk	<fs af='nahasa,unk,,,,,,'>
20	KuSagavAra	unk	<fs af='KuSagavAra,n,m,sg,3,d,0,0'>|<fs af='KuSagavAra,n,m,pl,3,d,0,0'>|<fs af='KuSagavAra,n,m,sg,3,o,0,0'>|<fs af='KuSagavAra,adj,any,any,,any,,'>
21	SOYla	unk	<fs af='SOYla,n,f,sg,3,d,0,0'>|<fs af='SOYla,n,f,sg,3,o,0,0'>
22	kaSIxAkArI	unk	<fs af='kaSIxAkArI,n,f,sg,3,d,0,0'>|<fs af='kaSIxAkArI,n,f,sg,3,o,0,0'>
23	raNaBUmi	unk	<fs af='raNaBUmi,n,f,sg,3,d,0,0'>|<fs af='raNaBUmi,n,f,sg,3,o,0,0'>
24	piCadZepana	unk	<fs af='piCadZepana,n,m,sg,3,d,0,0'>|<fs af='piCadZepana,n,m,pl,3,d,0,0'>|<fs af='piCadZepana,n,m,sg,3,o,0,0'>|<fs af='piCadZepana,n,m,pl,3,o,0,0'>
25	XImI	unk	<fs af='XImA,adj,f,any,,any,,'>
26	halakI	unk	<fs af='halakA,adj,f,any,,any,,'>|<fs af='halaka,v,f,sg,any,,yA,yA'>
27	sWAyI	unk	<fs af='sWAyI,adj,any,any,,any,,'>
28	valavalA	unk	<fs af='valavalA,n,m,sg,3,d,0,0'>|<fs af='valavalA,n,m,pl,3,d,0,0'>|<fs af='valavalA,n,m,sg,3,o,0,0'>|<fs af='valavalA,n,m,pl,3,o,0,0'>
